package.json

```diff
@@ -134,7 +134,7 @@
     "@commitlint/cli": "19.8.1",
     "@commitlint/config-conventional": "19.8.1",
     "@eslint/js": "9.32.0",
-    "@remcovaes/web-test-runner-vite-plugin": "1.2.2",
+    "@remcovaes/web-test-runner-vite-plugin": "github:overbalance/web-test-runner-vite-plugin#patch-1",
```

Comment:
I submitted this patch. This upgrade is not urgent, but I wanted to give us the option.